packages/core/echo/echo-db/src/proxy-db/load-object.ts

```diff
@@ -7,6 +7,14 @@ import { type EchoReactiveObject } from '@dxos/echo-schema';
 
 import { getObjectCore } from '../core-db';
 
+/**
+ * @param obj
+ */
+// TODO(burdon): Rename/review SDK.
+export const loadObject = <T extends EchoReactiveObject<any>>(obj: T): T => {
```

Comment:
is this expecting a reference to be passed in? if you passed in a object isn't it already loaded?

Reply:
I don't think this name makes sense. Let's discuss if this should be a part of the API

Reply:
see the use case below?